root/programs/C#/Frameworks/Infrastructure/CustomControl/PropertyGrid/CheckType.cs

```diff
@@ -64,21 +64,21 @@ public CheckType()
         #region プロパティ
 
         /// <summary>必須</summary>
-        private bool _isIndispensabile;
+        private bool _isIndispensable;
 
         /// <summary>必須</summary>
         [DefaultValue(false), NotifyParentProperty(true),
         RefreshProperties(RefreshProperties.Repaint)]
-        public bool IsIndispensabile
+        public bool IsIndispensable
```

Comment:
@yusukemaegawa 

>https://github.com/OpenTouryoProject/OpenTouryo/issues/148

This request was registered as a problem that has been granted to quality improvement label.

- [Problem of the properties of a custom control](https://github.com/OpenTouryoProject/OpenTouryo/issues/148)